src/pages/Controllers/Controllers.test.js

```diff
@@ -37,3 +37,43 @@ describe("Controllers page", () => {
     expect(wrapper.find("tbody tr").length).toBe(6);
   });
 });
+
+describe("Controllers overview panel", () => {
+  it("displays the machines count", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Controllers />
+        </Provider>
+      </MemoryRouter>
+    );
+    expect(wrapper.find(".overview__machines strong").text()).toBe(
+      "56 machines"
+    );
+  });
+  it("displays the applications count", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Controllers />
+        </Provider>
+      </MemoryRouter>
+    );
+    expect(wrapper.find(".overview__applications strong").text()).toBe(
```

Comment:
Thanks for the tests 👍 

You can probably just merge these assertions into the single test case :-)